apps/dashboard/src/app/select/select.component.html

```diff
@@ -1,41 +1,31 @@
 <div class="select"
      *ngIf="accounts && kbs">
-  <div class="select-container"
-       fxLayout="column"
-       fxFlex="0 0 auto">
+  <div class="select-container">
     <div class="logo">
-      <img src="assets/logos/logo_text.svg" />
+      <img src="assets/logos/logo_text.svg" alt="Nuclia"/>
     </div>
     <div class="select-content"
-         fxFlex="1 1 0"
          [@slideInOut]="selectKb">
       <div class="select-account"
-           *ngIf="!selectKb"
-           fxLayout="column">
-        <div class="select-account-list"
-             fxFlex="1 1 0">
+           *ngIf="!selectKb">
+        <div class="select-account-list">
```

Comment:
I used to use long class names, feel free to change it if you think they're to long.

Reply:
No worries, I like descriptive names more than short names